tests/htp-builder/rendered/rendered-customEndpoint.yaml

```diff
@@ -617,18 +617,14 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.uid
-            - name: STRAWS_COLLECTOR_POD_IP
```

Comment:
Should this be updated to `HTP_COLLECTOR_POD_IP` instead of deleting it?

Reply:
i suspect this was removed because it wasn't used, i could be wrong though